packages/extension/src/newtab/DndModal.tsx

```diff
@@ -1,95 +1,104 @@
-import React from 'react';
-import classnames from 'classnames';
-import {
-  getDefaultLink,
-  CustomTime,
-  TimeFormat,
-  getExpiration,
-  getTimeFormatExpiration,
-} from './dnd';
-import { Radio } from '../../../shared/src/components/fields/Radio';
-import { Dropdown } from '../../../shared/src/components/fields/Dropdown';
-import { TextField } from '../../../shared/src/components/fields/TextField';
-import { ModalCloseButton } from '../../../shared/src/components/modals/ModalCloseButton';
+import React, { FC, useState, useContext } from 'react';
+import { format } from 'date-fns';
+import { Radio } from '@dailydotdev/shared/src/components/fields/Radio';
+import { Dropdown } from '@dailydotdev/shared/src/components/fields/Dropdown';
+import { TextField } from '@dailydotdev/shared/src/components/fields/TextField';
+import { ModalCloseButton } from '@dailydotdev/shared/src/components/modals/ModalCloseButton';
 import {
   StyledModal,
   ModalProps,
-} from '../../../shared/src/components/modals/StyledModal';
-import styles from './DoNotDisturbModal.module.css';
+} from '@dailydotdev/shared/src/components/modals/StyledModal';
+import { getDefaultLink, dndOption, CustomTime, TimeFormat } from './dnd';
 import DndContext from './DndContext';
 
-export interface DoNotDisturbModalProps extends ModalProps {
-  defaultTimeFormat?: TimeFormat;
-}
-
-const timeFormatOptions = Object.values(TimeFormat).map((value) => ({
-  label: value,
-  value,
+const timeFormatOptions = Object.entries(dndOption).map(([k, v]) => ({
+  label: v.label,
+  value: k,
 }));
-
 const customTimeOptions = Object.values(CustomTime);
 
-const DoNotDisturbModal: React.FC<DoNotDisturbModalProps> = ({
-  defaultTimeFormat = TimeFormat.HALF_HOUR,
+const DoNotDisturbModal: FC<ModalProps> = ({
   onRequestClose,
   ...modalProps
 }) => {
-  const { setDndSettings, isActive } = React.useContext(DndContext);
-  const [link, setLink] = React.useState('');
-  const [customNumber, setCustomNumber] = React.useState(0);
-  const [customTimeIndex, setCustomTimeIndex] = React.useState(0);
-  const [dndTime, setDndTime] = React.useState(defaultTimeFormat);
+  const { dndSettings, setDndSettings, isActive } = useContext(DndContext);
+  const [link, setLink] = useState('');
```

Comment:
Default value should be null I think

Reply:
I thought of using an empty string so that the type will automatically be inferred instead of explicitly defining it. What do you think?

Reply:
I prefer null as it's more accurate for the purpose of empty values.

Reply:
True. Let's use null.